src/actions/guides/tutorial/00_overview.cr

```diff
@@ -0,0 +1,163 @@
+
+    We assume you have...
+
+    * some Web Development experience.
+    * worked with Crystal, or a similar language (e.g. Ruby, Python, etc...) before, or at least understand the Crystal syntax concepts somewhat. (i.e. classes, modules, instances)
+    * worked with some SQL database system before.
+    * followed the [Installing guide](#{Guides::GettingStarted::Installing.path}).
+    * installed Crystal. `crystal -v` should be at least #{LuckyCliVersion.min_compatible_crystal_version} or higher
+    * installed Lucky. `lucky -v` should be #{LuckyCliVersion.current_tag}.
+    * installed [Postgres](#{Guides::GettingStarted::Installing.path(anchor: Guides::GettingStarted::Installing::ANCHOR_POSTGRESQL)}). `psql --version` should be at least 10.x or higher
+    * installed [Node](#{Guides::GettingStarted::Installing.path(anchor: Guides::GettingStarted::Installing::ANCHOR_NODE)}). (used for building css and javascript)
+    * installed a [Process Manager](#{Guides::GettingStarted::Installing.path(anchor: Guides::GettingStarted::Installing::ANCHOR_PROC_MANAGER)}). (used for booting Webpack and Lucky at the same time)
+
+    ## The Wizard
+
+    From your terminal, we will start the Lucky wizard to generate our application. Run the `lucky init` command.
+
+    ```bash
+    lucky init
+    ```
+
+    This will ask us what "Project name" we would like to use. We will call it "clover" in all lowercase. Enter `clover`
+
+    ```
+    Project name?: clover
+    ```
+
+    Next, the wizard will ask us which format of application we would like to build. We are building a "full" app
+    which will allow us to use CSS, JavaScript, and have HTML views. Enter `full`
+
+    ```
+    API only or full support for HTML and Webpack? (api/full): full
+    ```
+
+    Lastly, the wizard would like to know if we want to generate authentication. We do! This will give us the ability
+    to create user accounts and login/out of our accounts. Enter `y`
+
+    ```
+    Generate authentication? (y/n): y
+    ```
+
+    Once that's done, Lucky will give you a few instructions on what to do next. We will follow those instructions
+    in the next step.
+
+    > For more information on the wizard, read the [Starting a Lucky Project](#{Guides::GettingStarted::StartingProject.path}) guide.
+
+    ## Booting Our Application
+
+    Before we boot our application, we still need to complete the setup process. This portion is specific to the
+    individual as only you will know your own databasae settings.
+
+    Let's start by changing the directory in to the project. Enter `cd clover`
+
+    ```bash
+    cd clover
+    ```
+
+    Now we need to make sure we can connect to Postgres so we can run SQL. Open up your Clover app project, and
+    open the `config/database.cr` file.
+
+    The default postgres connection uses the username `postgres`, and the password `postgres` along with the hostname `localhost`.
```

Comment:
I think it would connect with the `whoami` which isn't the default postgres. Like, if I do `psql`, it complains about not having a `jeremywoertink` user because when I installed postgres, it defaulted to `postgres`. Though, mine has no password, so I always have to update this config 😝

Reply:
I think that might depend on how you installed postgres. I have a `whoami` user -- my postgres is installed through brew, but I know Postgres.app exists as well.